lib/fog-local.rb

```diff
@@ -1 +0,0 @@
-require 'fog/local'
```

Comment:
You can automatically require the gem with `require 'fog/local'`.

When bundler runs it looks for the path `fog/local` instead of `fog-local` but i guess we could leve this file if you feel strong about it. This file is just not mandatory. =)